File: app/src/main/java/com/repair/proj/login/contract/RegistSecondContract.java
package com.repair.proj.login.contract;

import android.app.Activity;

import com.repair.proj.login.entity.SecondEnitity;
import com.repair.proj.nbase.NContract;


/**
 * 说明：
 * Created by code_nil on 2017/10/27.
 */

public interface RegistSecondContract {
    interface View extends NContract.View {
        void setLocationData(String data);

        SecondEnitity getRegistSecondData();
    }

    interface Presenter extends NContract.Presenter {
        void showLocationPicker(Activity activity);
    }

    interface Model extends NContract.Model {

    }
}

File: app/src/main/java/com/repair/proj/login/contract/RegistSecondContract.kt
package com.repair.proj.login.contract

import android.app.Activity
import com.repair.proj.base.Basefview

import io.xujiaji.xmvp.contracts.XContract

/**
 * Created by nie on 2017/10/23.
 */

interface RegistSecondContract {
    interface View : XContract.View {
    }

    interface Presenter : XContract.Presenter {
        fun showLocationPicker(activity: Activity)
    }

    interface Model : XContract.Model
}
